client/app/scripts/utils/color-utils.js

```diff
@@ -82,3 +82,13 @@ export function brightenColor(c) {
   }
   return color.toString();
 }
+
+export function darkenColor(c) {
+  let color = hsl(c);
+  if (hsl.l < 0.5) {
```

Comment:
Nope, blatantly copied this from the `brightenColor()` version. It made sense to me, if it's already on the darker/brighter side, go a bit slower towrads black/white.